parachain/pallets/inbound-queue/src/lib.rs

```diff
@@ -322,9 +327,14 @@ pub mod pallet {
 	/// API for accessing the delivery cost of a message
 	impl<T: Config> Get<BalanceOf<T>> for Pallet<T> {
 		fn get() -> BalanceOf<T> {
-			let pricing_parameters = T::PricingParameters::get();
-			let refund: BalanceOf<T> = T::WeightToFee::weight_to_fee(&T::WeightInfo::submit());
-			refund.saturating_add(pricing_parameters.rewards.local)
+			let weight_fee = T::WeightToFee::weight_to_fee(&T::WeightInfo::submit());
+			let length_fee = T::LengthToFee::weight_to_fee(&Weight::from_parts(
+				T::MaxMessagePayloadSize::get() as u64,
+				0,
+			));
+			let refund = weight_fee.saturating_add(length_fee);
+			let reward = T::PricingParameters::get().rewards.local;
+			refund.saturating_add(reward)
```

Comment:
I think we should break this out into a function named for example `Pallet::calculate_delivery_cost` which takes a `length` parameter.

Then we can compute the worst-case cost and actual cost.

Note that you also need to use this function when rewarding the relayer.

Reply:
https://github.com/Snowfork/snowbridge/pull/1065/commits/0eaa53fb013fff61bb5cbd65a41e3b7adcd91d69